.well-known/security.txt

```diff
@@ -0,0 +1,5 @@
+Contact: mailto:security@phpmyadmin.net
```

Comment:
I don't know if we keep `Contact: mailto:security@phpmyadmin.net`
or change it to `Contact: https://www.phpmyadmin.net/security/`.

Reply:
I would use the URL but the email seems more future proof

Reply:
But more likely to be scraped by bots for spamming ..